shields.example.env

```diff
@@ -0,0 +1,19 @@
+# Base URL for redirects etc.
+BASE_URL=http://localhost:8080
+# Where your homepage is
+INFOSITE=http://localhost:8080/try.html
```

Comment:
Not always. In production, `INFOSITE` is `https://shields.io` and `BASE_URL` is `https://img.shields.io`, afaik.